guides/source/active_storage_overview.md

```diff
@@ -1043,20 +1042,16 @@ No CORS configuration is required for the Disk service since it shares your app
 [
   {
     "AllowedHeaders": [
-      "*"
+      "Content-Type",
+      "Content-MD5",
+      "Content-Disposition"
     ],
     "AllowedMethods": [
       "PUT"
     ],
     "AllowedOrigins": [
       "https://www.example.com"
     ],
-    "ExposeHeaders": [
-      "Origin",
-      "Content-Type",
-      "Content-MD5",
-      "Content-Disposition"
-    ],
```

Comment:
Should we be removing `ExposeHeaders` completely from the example configuration here? Some of these are irrelevant to a response header, as you've mentioned, but others like `Content-Type` are valid (albeit default / safelisted) response headers for Amazon S3. Might be worth keeping `ExposeHeaders` and pulling an example response header from https://docs.aws.amazon.com/AmazonS3/latest/API/RESTCommonResponseHeaders.html?

Reply:
ActiveStorage JS doesn't access response headers. So `ExposeHeaders` is not required at all.
If it supports [multi partupload](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html) in browsers for uploading large files, `ExposeHeaders` should contain `ETag`. But it doesn't support multipart upload now.